simple-wp-helpdesk/includes/class-email.php

```
@@ -159,6 +169,158 @@ function swh_wrap_html_email( $body, $attachments = array() ) {
+	$from    = isset( $params['sender'] ) && is_string( $params['sender'] ) ? $params['sender'] : (
+		isset( $params['from'] ) && is_string( $params['from'] ) ? $params['from'] : ''
+	);
+
+	// Extract email address from "Name <email@example.com>" format.
+	if ( preg_match( '/<([^>]+)>/', $from, $m ) ) {
+		$from = $m[1];
+	}
+	$from = sanitize_email( trim( $from ) );
+
+	// Parse ticket ID from subject: [TKT-XXXX].
+	if ( ! preg_match( '/\[TKT-(\d+)\]/i', $subject, $tm ) ) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $tm. Configured minimum length is 3. (undefined)

(ShortVariable)